package.json

```diff
@@ -165,5 +165,11 @@
     },
     "engines": {
         "npm": ">=8.11.0"
+    },
+    "overrides": {
+        "react-ticker": {
+            "react": "^18.2.0",
+            "react-dom": "^18.2.0"
+        }
```

Comment:
@niloofar-deriv I have a doubt,Why is this over ride on react-ticker?

Reply:
check this pls: https://github.com/AndreasFaust/react-ticker/issues/60#issuecomment-1263575098